### PR TITLE
Change button`s 'Save' behavior. After click on it, redirect to user`…

### PR DESCRIPTION
**Pull request by lisovskyvlad**
Hi Dmitry, I love this project, especially, stack you choose. I would like to learn it and your project seems to be good point to do it. 

So, in most of apps I developed (typically, it is server-render style app), after you submit form with model, then redirect is following to list/table of all models. What do you think?

### CODE REVIEW

File: src/scripts/components/users/Edit.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ const UserEdit = React.createClass({
 
     if (id) {
       UserActions.update(id, data, this.onSave);
+      this.transitionTo('users');
     } else {
       UserActions.create(data, this.onSave);
```
Comment (Semigradsky) [inline src/scripts/components/users/Edit.js:48]: Maybe redirect to list after creating user too?

File: src/scripts/components/users/Edit.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import React from 'react';
+import Router from 'react-router';
```
Comment (Semigradsky) [inline src/scripts/components/users/Edit.js]: `import { Navigation } from 'react-router';` looks more nice)

File: src/scripts/components/users/Edit.js (hunk 1 above)
Comment (lisovskyvlad) [inline src/scripts/components/users/Edit.js:48]: Yeah, and better to show some alert.

**Comment by Semigradsky** [comment]
Great! I am glad that you are interested in this project

**Pull request merged**